docs/UserGuide.md

```diff
@@ -611,10 +632,17 @@ for the **List** command, the `INDEX` argument can be either a `Group` or a `Per
 </div>
 
 ### General
+Refer to the details of the general commands [here](#general).
```

Comment:
This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.

Reply:
fixed, please check